src/components/Button.tsx

```diff
@@ -7,16 +7,31 @@ export function Button({
   primaryText,
   secondaryText,
   handleClick,
+  type,
+  ariaDescription,
+  initialFocus,
 }: {
   /** The button primary text */
   primaryText: string;
   /** The button secondary text */
   secondaryText?: string;
   /** Parent handler for the form */
   handleClick: JSX.MouseEventHandler<HTMLButtonElement> | undefined;
+  /** Optional button type */
+  type?: 'submit' | 'reset' | 'button';
+  /** Aria button description */
+  ariaDescription?: string;
+  /** Whether to autofocus this button */
+  initialFocus?: true;
 }): JSX.Element {
   return (
-    <button className="button" onClick={handleClick}>
+    <button
+      className="button"
+      onClick={handleClick}
+      type={type}
```

Comment:
Does this have a natural HTML default, or should we perhaps default it to "button"?

Reply:
it has a default of button, yep